SPECS/interface-rename.spec

```diff
@@ -10,6 +10,8 @@ Source0: https://code.citrite.net/rest/archive/latest/projects/XS/repos/interfac
 
 Provides: gitsha(https://code.citrite.net/rest/archive/latest/projects/XS/repos/interface-rename/archive?at=v2.0.2&format=tar.gz&prefix=interface-rename-2.0.2#/interface-rename-2.0.2.tar.gz) = 76ee8cb29338cf9ee113ff07222d623e177ffddb
 
+# XCP-ng patches
+Patch1000:      interface-rename-2.0.2-ExecStartPre-udevadm.patch
```

Comment:
Please add a newline after this line so that our `#XCP-ng patches` section is clearly separated from the rest. For readability and also to lower the chances of merge conflicts when we update with the next version of that package from next Citrix Hypervisor.

I think you also forgot about the patch renaming.